bgpd/bgp_nb_config.c

```diff
@@ -87,12 +87,12 @@ int bgp_router_create(struct nb_cb_create_args *args)
 	case NB_EV_APPLY:
 		vrf_dnode = yang_dnode_get_parent(args->dnode,
 						  "control-plane-protocol");
-		vrf = nb_running_get_entry(vrf_dnode, NULL, true);
+		vrf_name = yang_dnode_get_string(vrf_dnode, "./vrf");
 
```

Comment:
dont you have to register for bgp too ?

Reply:
No. This one was the only place where it was using the vrf. Now it just gets the name from control-plane-protocol dnode, it doesn't need the actual vrf to be created.
More than that, we don't want to delete the "router bgp" configuration when the vrf is deleted.

Reply:
your answer is ok for me. thanks.
ok, I realise that rpki plugin has been magically ignored from the nb config porting.